### DISCUSSION TITLE
Uno.Wasm.Bootstrap.Server only works on Publish?

### DISCUSSION THREAD

**Discussion by SerratedSharp** [unoplatform/Uno.Wasm.Bootstrap#751] [Q&A]
I saw in release notes about the Uno.Wasm.Bootstrap.Server package for integration with ASP.NET, and went through this with version 7:
https://platform.uno/docs/articles/external/uno.wasm.bootstrap/doc/deploy-and-publish.html#integration-with-aspnet-core

The setup process is nice, but **as far as I can tell it only copies the packages into wwwroot during _publish_.  So this means when doing just a local build to launch the ASP.NET project, then it is not getting included.  Is this expected or have I misunderstood?**

**Comment by jeromelaban** [comment]
Thanks for the report! Could you provide a [binlog of the build](https://platform.uno/docs/articles/uno-builds-troubleshooting.html), even if it does not create build errors? The `Uno.Wasm.Bootstrap.Server` package is a standard library and should be included in the output like any other dependency.

**Reply by SerratedSharp** [reply]
Will followup with those details.  Note I didn't know about embedded mode prior to this posting, which should address other issues I mentioned in earlier edit, with referencing/loading the package since the embedded.js lives outside the package folder.

**Reply by SerratedSharp** [reply]
@jeromelaban  I have zipped projects with binlogs for both the WASM and ASP.NET project.  Can I send via Discord?

**Reply by jeromelaban** [reply]
@SerratedSharp Thanks. Unless it's private source, it's best to publish the binlog here. It's even better if you can provide a repro independent of your code.

**Reply by SerratedSharp** [reply]
Ok I wasn't sure based on comments in docs how much system information binlog was including.  Attached is the reproduction with binlog's in respective folders for each project created with `msbuild /r /bl project.csproj`  

Note I had the issue initially with .NET 7.  Since then I happened to have installed the .NET 8 SDK preview, but the projects target 7.  It looks like the powershell developer command prompt was using the .NET 8 SDK during build just FYI in case that impacts anything:
![image](https://github.com/unoplatform/Uno.Wasm.Bootstrap/assets/97156524/f10b79fa-d72f-4ae8-bbde-74df4d3a1ade)
[WasmNotCopyOnBuild.zip](https://github.com/unoplatform/Uno.Wasm.Bootstrap/files/12361746/WasmNotCopyOnBuild.zip)
